src/main/java/com/cloudbees/jenkins/plugins/docker_build_env/Docker.java

```diff
@@ -223,6 +255,10 @@ public String runDetached(String image, String workdir, Map<String, String> volu
         }
         args.add(image).add(command);
 
+        if(sudo) {
```

Comment:
Would be nice to generalize the code a bit to follow the DRY principle. Maybe a utility method?